src/lib/core/redux/slices/rtcConnection/index.ts

```diff
@@ -307,6 +307,36 @@ export const selectIsAcceptingStreams = (state: RootState) => state.rtcConnectio
 /**
  * Reactors
  */
+startAppListening({
+    predicate: (_action) => {
```

Comment:
```suggestion
    predicate: (action) => {
```

Reply:
I'll keep it for now as that's been the convention elsewhere, can overhaul later perhaps?

Reply:
We used it to mark that the variable was unused. But we should rename them to just `_` instead of `_action` to make it clearer. No worries. I can clean this up 👍